pkg/kubelet/kubelet_pods.go

```diff
@@ -2072,6 +2072,17 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 		if oldStatus.Resources == nil {
 			oldStatus.Resources = &v1.ResourceRequirements{}
 		}
+
+		convertCustomResources := func(inResources, outResources v1.ResourceList) {
+			for resourceName, resourceQuantity := range inResources {
+				if resourceName == v1.ResourceCPU || resourceName == v1.ResourceMemory ||
+					resourceName == v1.ResourceStorage || resourceName == v1.ResourceEphemeralStorage {
+					continue
+				}
+				outResources[resourceName] = resourceQuantity
```

Comment:
should this be a `.DeepCopy()`?

Reply:
Good catch. It should be DeepCopy for consistency.

Reply:
Done! Thanks for catching that!